Add Timelab suborg info

matteobachetti opened the pull request.

dpshelio commented on _projects/2018/timelab/quasiperiodic.md:
```diff
@@ -0,0 +1,56 @@
+
++ [Ingram et al. 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.461.1967I&link_type=EJOURNAL)
+
++ [Stevens & Uttley 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2796S&link_type=ARTICLE)
```
Both links are not open access, I would suggest to also provide the link to ArXiv:
```md
+ Ingram et al. 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.461.1967I&link_type=EJOURNAL) ([open-access preprint](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.461.1967I&link_type=PREPRINT)
+ [Stevens & Uttley 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2796S&link_type=ARTICLE) ([open-access preprint](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2796S&link_type=PREPRINT)
```

abigailStev replied:
The arxiv is accessible via ADS.

dpshelio commented on _projects/2018/timelab/quasiperiodic.md:
```diff
@@ -0,0 +1,56 @@
+issues:
+ - https://github.com/StingraySoftware/stingray/issues/293
+mentors:
+ - matteobachetti
```
Will also @abigailStev mentoring this? we prefer 2 mentor per projects if possible (one as a backup)

matteobachetti replied:
She will actually be the main mentor, I think. I just put my name as a placeholder :)

dpshelio commented on _includes/members/Timelab.html:
```diff
@@ -0,0 +1,20 @@
+                  <path fill="#333333" d="{{ site.gh_logo }}"/>
+                </svg>
+              </span>
+	      Timelab</a>
```
Maybe you want also to add the slack or any mailing list?

matteobachetti replied:
Done

dpshelio commented on _config.yml:
```diff
@@ -56,6 +56,9 @@ ims_repository: https://bitbucket.org/italianmarssociety/eras/src
 coin_repository: https://github.com/COINtoolbox/
 
+timelab_repository: https://github.com/StingraySoftware/
+timelab_url: http://www.timelabtechnologies.com/ideas.html
```
Should this be the `home` url, instead of the ideas from last year?

dpshelio commented on _projects/2018/timelab/quasiperiodic.md:
```diff
@@ -0,0 +1,56 @@
+
+Both methods can actually be used for pulsars too (it is the trivial case!).
+
+#### Deliverables
```
We are also suggesting to break the project into milestones for the three evaluation periods. That helps the students to create better applications.

matteobachetti replied:
Done

matteobachetti commented:
@dpshelio thanks for the review! I addressed your comments, please let me know if there is more to change.
When this project is fully reviewed, we  will add a few more.
